docs/Authentication.md

```diff
@@ -249,6 +249,7 @@ const MyLoginPage = ({ theme }) => {
             <form onSubmit={submit}>
                 <input name="email" type="email" value={email} onChange={e => setEmail(e.target.value)} /> 
                 <input name="password" type="password" value={password} onChange={e => setPassword(e.target.value)} />
+                <Notification />
```

Comment:
Probably nitpicking but I would put the `<Notification />` outside the form. I'll let the other core members decide though